templates/common/on-prem/files/keepalived.yaml

```diff
@@ -97,7 +97,23 @@ contents:
             done
           }
 
+          remove_vip()
+          {
+            address=$1
+            netmask=32
+            if [[ "$address" =~ : ]]; then
+                netmask=128
+            fi
+            interface=$(ip -o a | grep $address | awk '{print $2}')
+            if [ -n "$interface" ]; then
+                ip a del ${address}/${netmask} dev $interface
+            fi
+          }
+
           set -ex
+          # Ensure that we don't have stale VIPs configured
```

Comment:
If we intend to someday remove the workaround then we should reference the upstream keepalived issue. I personally don't see a problem keeping the removal of the VIP in the startup script forever, so up to you if you want to modify the patch.

Reply:
In the interest of not throwing away the ci passes I have I think I'll leave it alone for now. I can push a small followup to add a reference to the bz that can be merged whenever. If that doesn't make 4.8 it won't be a big deal.

Reply:
@cybertron Maybe we should add that now, since it had to be rebased in the last 15 minutes anyway?

Reply:
Yep, good point. Thanks for the reminder.